packages/framework/src/Resources/views/Debug/Elasticsearch/template.html.twig

```diff
@@ -0,0 +1,139 @@
+            <strong>Elasticsearch</strong>
+            <span class="count">
+                <span>{{ collector.data.requestsCount }}</span>
+            </span>
+        </span>
+    {% endif %}
+{% endblock %}
+
+{% block panel %}
+    <style type="text/css">
+        .status--ok {
+            background-color: #DDFFDD;
+        }
+        .status--404 {
+            background-color: #FFFFDD;
+        }
+        .status--error {
+            background-color: #FFDDDD;
+        }
+    </style>
+
+    <h1>Elasticsearch - total {{ collector.data.requestsCount }} queries in {{ '%0.2f'|format(collector.data.totalRequestsTime * 1000) }} ms</h1>
+
+    {% for index, request in collector.data.requests %}
+        {% if request.statusCode is null or request.statusCode >= 500 %}
+            {% set statusClass = 'status--error' %}
+        {% elseif request.statusCode >= 400 %}
+            {% set statusClass = 'status--404' %}
+        {% else %}
+            {% set statusClass = 'status--ok' %}
+        {% endif %}
+        <table>
+            <tr style="">
+                <th class="{{ statusClass }}" style="width: 28px;">{{ request.statusCode }}</th>
+                <th class="{{ statusClass }}" style="width: 70px;">{{ request.method }}</th>
+                <th class="{{ statusClass }}">{{ request.uri }}</th>
+                <td class="{{ statusClass }}" style="width: 200px;">
+                    {% if request.requestJson is not null %}
+                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-response-data-{{ index }}').style.display = ''; return false;">
+                            Response
+                        </button>
+                    {% endif %}
+                    {% if request.requestData is not null %}
+                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-request-data-{{ index }}').style.display = ''; return false;">
+                            Request
+                        </button>
+                    {% endif %}
+                    {% if request.response is not null %}
+                        <button type="button" style="float: right;" onclick="this.style.display = 'none'; document.getElementById('js-elasticsearch-request-json-{{ index }}').style.display = ''; return false;">
+                            Json
+                        </button>
+                    {% endif %}
+                </td>
+            </tr>
+            <tr>
+                <td colspan="4" style="padding: 5px; word-break: break-all;">
+                    {{ request.requestCurl  }}
+                </td>
+            </tr>
+
+            {% if request.requestJson is not null %}
+                <tr id="js-elasticsearch-request-json-{{ index }}" style="display: none;">
+                    <td colspan="4"><pre>{{ request.requestJson }}</pre></td>
```

Comment:
is it possible to dump also json data?

Reply:
Why? Using of function `dump()` writes mess around like string length etc... Tag <pre> is exact what I needed for readable and copy-paste into Kibana for example...
So, It is my point of view, If SSFW team preferes dump, you can.

Reply:
ok, if that is some input into tools like KIBANA that is OK for me